PR#3447 – Liquid Exception

- [ ] Liquid Exception: Could not locate the included file 'classifyEnergyETF.html' in any of ["/home/runner/work/blog/blog/_includes", "/home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/minima-2.5.2/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in /home/runner/work/blog/blog/_posts/2026/04/2026-04-06-20260406ClosingIndexes.md
```
Run bundle exec jekyll build --trace --incremental --baseurl "/blog"
  bundle exec jekyll build --trace --incremental --baseurl "/blog"
  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_PAGES: true
    JEKYLL_ENV: production
    JEKYLL_GITHUB_TOKEN: 
    LOG_LEVEL: debug
  
To use retry middleware with Faraday v2.0+, install `faraday-retry` gem
Configuration file: /home/runner/work/blog/blog/_config.yml
            Source: /home/runner/work/blog/blog
       Destination: /home/runner/work/blog/blog/_site
 Incremental build: enabled
      Generating... 
          Skipping: _posts/2029/01/2029-01-20-ByeByeDonnie.md has a future date
  AI Related Posts: Creating cache [.ai_related_posts_cache.sqlite3]
  AI Related Posts: Fetch disabled. Using cached related posts data.
  AI Related Posts: Found 0 cached embeddings; skipped 1685 fetches
       Jekyll Feed: Generating feed for posts
         AutoPages: Disabled/Not configured in site.config.
        Pagination: ----------------------------
        Pagination: Page: blogpager.html
        Pagination:  Active configuration
        Pagination:   Enabled:          true
        Pagination:   Items per page:   15
        Pagination:   Permalink:        /page/:num/
        Pagination:   Title:            :title - page :num of :max
        Pagination:   Limit:            0
        Pagination:   Sort by:          date
        Pagination:   Sort reverse:     true
        Pagination:  Active Filters
        Pagination:   Collection:       posts
        Pagination:   Offset:           0
        Pagination:   Category:         [Not set]
        Pagination:   Tag:              [Not set]
        Pagination:   Locale:           [Not set]
        Pagination:  Filtering by: Category  1685 => 1685
        Pagination:  Filtering by: Tag       1685 => 1685
        Pagination:  Filtering by: Locale    1685 => 1685
        Pagination: Rolling through the date fields for all documents
        Pagination: Complete, processed 1 pagination page(s)
  Liquid Exception: Could not locate the included file 'classifyEnergyETF.html' in any of ["/home/runner/work/blog/blog/_includes", "/home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/minima-2.5.2/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in /home/runner/work/blog/blog/_posts/2026/04/2026-04-06-20260406ClosingIndexes.md
bundler: failed to load command: jekyll (/home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/bin/jekyll)
/home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/tags/include.rb:219:in 'Jekyll::Tags::OptimizedIncludeTag#locate_include_file': Could not locate the included file 'classifyEnergyETF.html' in any of ["/home/runner/work/blog/blog/_includes", "/home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/minima-2.5.2/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. (IOError)
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/tags/include.rb:201:in 'Jekyll::Tags::OptimizedIncludeTag#render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/liquid-4.0.4/lib/liquid/block_body.rb:103:in 'Liquid::BlockBody#render_node_to_output'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/liquid-4.0.4/lib/liquid/block_body.rb:91:in 'Liquid::BlockBody#render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/liquid-4.0.4/lib/liquid/template.rb:206:in 'block in Liquid::Template#render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/liquid-4.0.4/lib/liquid/template.rb:240:in 'Liquid::Template#with_profiling'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/liquid-4.0.4/lib/liquid/template.rb:205:in 'Liquid::Template#render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/liquid-4.0.4/lib/liquid/template.rb:218:in 'Liquid::Template#render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:39:in 'block (3 levels) in Jekyll::LiquidRenderer::File#render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:59:in 'Jekyll::LiquidRenderer::File#measure_counts'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:38:in 'block (2 levels) in Jekyll::LiquidRenderer::File#render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:63:in 'Jekyll::LiquidRenderer::File#measure_bytes'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:37:in 'block in Jekyll::LiquidRenderer::File#render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:70:in 'Jekyll::LiquidRenderer::File#measure_time'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/liquid_renderer/file.rb:36:in 'Jekyll::LiquidRenderer::File#render!'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:129:in 'Jekyll::Renderer#render_liquid'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:80:in 'Jekyll::Renderer#render_document'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/renderer.rb:63:in 'Jekyll::Renderer#run'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:572:in 'Jekyll::Site#render_regenerated'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:557:in 'block (2 levels) in Jekyll::Site#render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:556:in 'Array#each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:556:in 'block in Jekyll::Site#render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:555:in 'Hash#each_value'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:555:in 'Jekyll::Site#render_docs'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:210:in 'Jekyll::Site#render'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:80:in 'Jekyll::Site#process'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:28:in 'Jekyll::Command.process_site'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:65:in 'Jekyll::Commands::Build.build'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:36:in 'Jekyll::Commands::Build.process'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in 'block in Jekyll::Command.process_with_graceful_fail'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in 'Array#each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in 'Jekyll::Command.process_with_graceful_fail'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:18:in 'block (2 levels) in Jekyll::Commands::Build.init_with_program'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in 'block in Mercenary::Command#execute'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in 'Array#each'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in 'Mercenary::Command#execute'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in 'Mercenary::Program#go'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/mercenary-0.4.0/lib/mercenary.rb:21:in 'Mercenary.program'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/gems/jekyll-4.4.1/exe/jekyll:15:in '<top (required)>'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/rubygems.rb:303:in 'Kernel#load'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/rubygems.rb:303:in 'Gem.activate_and_load_bin_path'
	from /home/runner/work/blog/blog/vendor/bundle/ruby/4.0.0/bin/jekyll:25:in '<top (required)>'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/bundler/cli/exec.rb:61:in 'Kernel.load'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/bundler/cli/exec.rb:61:in 'Bundler::CLI::Exec#kernel_load'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/bundler/cli/exec.rb:24:in 'Bundler::CLI::Exec#run'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/bundler/cli.rb:500:in 'Bundler::CLI#exec'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/bundler/vendor/thor/lib/thor/command.rb:28:in 'Bundler::Thor::Command#run'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in 'Bundler::Thor::Invocation#invoke_command'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/bundler/vendor/thor/lib/thor.rb:538:in 'Bundler::Thor.dispatch'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/bundler/cli.rb:35:in 'Bundler::CLI.dispatch'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/bundler/vendor/thor/lib/thor/base.rb:584:in 'Bundler::Thor::Base::ClassMethods#start'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/bundler/cli.rb:29:in 'Bundler::CLI.start'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/gems/4.0.0/gems/bundler-4.0.3/exe/bundle:28:in 'block in <top (required)>'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/bundler/friendly_errors.rb:118:in 'Bundler.with_friendly_errors'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/gems/4.0.0/gems/bundler-4.0.3/exe/bundle:20:in '<top (required)>'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/rubygems.rb:303:in 'Kernel#load'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/lib/ruby/4.0.0/rubygems.rb:303:in 'Gem.activate_and_load_bin_path'
	from /opt/hostedtoolcache/Ruby/4.0.1/x64/bin/bundle:25:in '<main>'
Error: Process completed with exit code 1.
```

_Originally posted by @RalphHightower in https://github.com/RalphHightower/blog/issues/3447#issuecomment-4195455219_

--- a/_includes/classifyEnergyETF.html
+++ b/_includes/classifyEnergyETF.html
@@ -1,0 +1,98 @@
+### ⚡ Energy ETF Classification (Clean, Rational, Archive‑Ready)
+
+#### 🟦 1. MLP / Pipeline & Midstream (Income + Infrastructure)
+
+These behave differently from pure energy equities.
+
+| Symbol | Name |
+|--------|------|
+| AMLP | Alerian MLP ETF |
+| AMZA | InfraCap MLP ETF |
+| ENFR | ALPS Alerian Energy Infrastructure ETF |
+| MLPX | Global X MLP & Energy Infrastructure ETF |
+| TPYP | Tortoise North American Pipeline ETF |
+
+Behavior:
+- High yield
+- Midstream focus
+- Less sensitive to crude price swings
+- Often K‑1 or K‑1‑free variants
+
+#### 🟩 2. Broad U.S. Energy Sector ETFs
+These are the core “Energy behaves like a region” funds.
+
+| Symbol | Name |
+|--------|------|
+| FENY | Fidelity MSCI Energy Index ETF |
+| IYE | iShares U.S. Energy ETF |
+| VDE | Vanguard Energy Index Fund ETF |
+| XLE | Energy Select Sector SPDR ETF |
+
+Behavior:
+- Track U.S. majors (XOM, CVX)
+- Good for regional‑style reading
+- Strong candidates for your daily snapshot
+
+#### 🟧 3. Exploration & Production (E&P) ETFs
+Higher beta, more sensitive to crude.
+
+| Symbol | Name |
+|--------|------|
+| IEO | iShares U.S. Oil & Gas Exploration & Production ETF |
+| PXE | Invesco Energy Exploration & Production ETF |
+| XOP | SPDR S&P Oil & Gas Exploration & Production ETF |
+
+Behavior:
+- Strong macro responsiveness
+- Excellent for “firehose” days
+- Good doctrinal signals
+
+#### 🟥 4. Oil Services / Equipment
+These track drillers, service firms, and equipment providers.
+
+| Symbol | Name |
+|--------|------|
+| OIH | VanEck Oil Services ETF |
+
+Behavior:
+- Very cyclical
+- High volatility
+- Moves on capex cycles
+
+#### 🟪 5. Leveraged Energy ETFs (2× Daily)
+These are not suitable for long‑term tracking but useful for signal amplification.
+
+| Symbol | Name |
+|--------|------|
+| ERX | Direxion Daily Energy Bull 2X ETF |
+| GUSH | Direxion Daily S&P Oil & Gas E&P Bull 2X ETF |
+| TEXU | Direxion Daily Energy Top 5 Bull 2X ETF |
+
+Behavior:
+- Daily reset
+- Not doctrinal
+- Should not be in your core table
+- But can be a “volatility indicator” module if you want one
+
+#### 🌍 6. Global Energy
+Broader than U.S. majors.
+
+| Symbol | Name |
+|--------|------|
+| IXC | iShares Global Energy ETF |
+
+Behavior:
+- Includes Shell, BP, TotalEnergies
+- Good for cross‑regional comparison
+
+#### 🟨 7. Crude‑Linked / Futures‑Based
+These track crude price exposure, not equities.
+
+| Symbol | Name |
+|--------|------|
+| OILK | ProShares K‑1 Free Crude Oil ETF |
+
+Behavior:
+- Tracks WTI futures
+- Useful for doctrine but not equity correlation
+

